recipes/epicscorelibs/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "epicscorelibs" %}
+{% set version = "7.0.2.99.0.0" %}
```

Comment:
This version number is pretty long. Did you check that conda can handle it properly?

Reply:
It is PEP440 compatible so I think it should be ok. I will look at the build log. Other than looking there, do you have other suggestions on how to test this?

Reply:
https://github.com/mdavidsaver/epicscorelibs/blob/9a95fe4c0d20eb96a7adc8a5789e50dad75c6255/setup.py#L26-L29

Reply:
The build log shows that conda handled it beautifully